.travis.yml

```diff
@@ -13,28 +13,63 @@ matrix:
   - python: 2.7
     env: CONDA_ENV=py27-min
   - python: 2.7
+    env: CONDA_ENV=py27-min SKIP_CI=--skip-optional-ci
```

Comment:
@shoyer-- This is the intent of my new changes: We really only need to test the h5netcdf tests that are now marked as flakey.  I added this extra tests and make the rest of the tests to skip optional CI if they aren't marked as allowed failures.